gene2phenotype_project/gene2phenotype_app/serializers.py

```diff
@@ -1104,16 +1110,42 @@ def create(self, validated_data):
                     raise serializers.ValidationError({"message": f"Data already submited to G2P '{lgd_obj.stable_id.stable_id}'"})
                 else:
                     raise serializers.ValidationError({"message": f"This is an old G2P record '{lgd_obj.stable_id.stable_id}'"})
+        
+        else:
+            raise serializers.ValidationError({"message" : "To publish a curated entry, locus, allelic requirement and disease are neccessary"})
```

Comment:
In this part, when you fetch the LGD entry you can convert these queries into one:
```
lgd_obj = LocusGenotypeDisease.objects.filter(locus__name=locus,
                                          genotype__value=allelic_requirement,
                                          disease__name=disease)
```

Reply:
Yes did not make any changes to this bit of the existing code, i think it is for publish part but will also be done differently, just added comments to it so I can remember to make changes to it in the future

Reply:
You can updated some of these things now.